enrichissement.py

```diff
@@ -49,10 +49,110 @@ def main():
     df = reorganisation(df)
 
     df = enrichissement_geo(df)
+    
+    df = enrichissement_type_entreprise(df)
 
+    df = apply_luhn(df)
+
+    df = manage_column_final(df)
+    
     df.to_csv("decp_augmente.csv", quoting=csv.QUOTE_NONNUMERIC)
 
 
+def manage_column_final(df): 
+    """Rename de certaines colonne et trie des colonnes"""
+    df = df.rename(columns = {
+                     'montant' : 'montantCalcule',
+                     "natureObjet": "natureObjetMarche",
+                     "categorieEntreprise" : "categorieEtablissement"
+                     })
+    
+    #df = df.drop(columns = ['latitudeCommuneEtablissement', 'longitudeCommuneEtablissement', 'latitudeCommuneAcheteur', 'longitudeCommuneAcheteur'])
+
+    df = df[['id', 'source', 'type', 'natureObjetMarche', 'objetMarche', 'codeCPV', 'referenceCPV', 'dureeMois',
+        'dateNotification', 'anneeNotification', 'moisNotification', 'dureeMoisEstime',
+        'dureeMoisCalculee', 'datePublicationDonnees','montantOriginal', 'montantEstime', 'montantCalcule', 'nbTitulairesSurCeMarche',
+        'formePrix', 'lieuExecutionCode', 'lieuExecutionTypeCode',
+        'lieuExecutionNom', 'nature', 'procedure',
+        'idAcheteur', 'verifSirenAcheteur', 'nomAcheteur', 'codeRegionAcheteur', 'regionAcheteur', 'codePostalAcheteur', 'libelleCommuneAcheteur', 'codeCommuneAcheteur',
+        'superficieCommuneAcheteur', 'populationCommuneAcheteur', 'geolocCommuneAcheteur',
+        'typeIdentifiantEtablissement',
+        'siretEtablissement', 'sirenEtablissement', 'nicEtablissement', 'verifSirenEtablissement', "categorieEtablissement", 'denominationSocialeEtablissement',
+        'adresseEtablissement', 'communeEtablissement', 'codeCommuneEtablissement', 'codePostalEtablissement',
+        'codeTypeEtablissement', 
+        'superficieCommuneEtablissement', 'populationCommuneEtablissement',
+        'distanceAcheteurEtablissement', 
+        'geolocCommuneEtablissement']]
+    return df
```

Comment:
Ce serait beaucoup plus clair IMHO de passer par une variable intermédiaire dans le genre :
```suggestion
    columns_to_keep = [col1, col2,  ...]
    df = df[cols]
    return df
```

Reply:
Idéalement, passer une conf, ou peut être une "constante" définie au début du script ?